crates/header-translator/src/main.rs

```diff
@@ -129,14 +132,51 @@ fn main() {
     cache.update(&mut final_result);
     drop(span);
 
-    for (library_name, files) in final_result.libraries {
+    for (library_name, files) in &final_result.libraries {
         let _span = info_span!("writing", library_name).entered();
-        let output_path = crate_src.join("generated").join(&library_name);
+        let output_path = crate_src.join("generated").join(library_name);
```

Comment:
Can we add this line so that the generator doesn't error if the library dir hasn't been manually created yet?

```rust
std::fs::create_dir_all(&output_path)?;
```

Reply:
Huh, I thought I included that from your changes before.

I've included it now, at least